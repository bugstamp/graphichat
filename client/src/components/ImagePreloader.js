import React from 'react';

import loginBgImage from '../assets/images/login-bg_1920_80.jpg';
import regBgImage from '../assets/images/reg-bg__1920_65.jpg';

const ImagePreloader = () => (
  <div className="image-preloader" style={{ display: 'none' }}>
    <img src={loginBgImage} alt="" />
    <img src={regBgImage} alt="" />
  </div>
);

export default ImagePreloader;
