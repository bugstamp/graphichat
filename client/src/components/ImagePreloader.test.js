import React from 'react';
import { shallow } from 'enzyme';

import ImagePreloader from './ImagePreloader';

describe('test ImagePreloader.js', () => {
  test('snapshot testing', () => {
    const wrapper = shallow(<ImagePreloader />);

    expect(wrapper).toMatchSnapshot();
  });
});
