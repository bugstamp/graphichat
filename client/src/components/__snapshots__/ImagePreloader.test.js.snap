// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ImagePreloader.js snapshot testing 1`] = `
<div
  className="image-preloader"
  style={
    Object {
      "display": "none",
    }
  }
>
  <img
    alt=""
    src="test-file-mock"
  />
  <img
    alt=""
    src="test-file-mock"
  />
</div>
`;
